### ISSUE TITLE
Please add new IAM permissions for Cluster Autoscaler to enable Instance Requirements support

### ISSUE THREAD

**Issue by herrbpl** [terraform-aws-modules/terraform-aws-iam#346]
## Is your request related to a problem? Please describe.
Current version is missing permissions to allow actions for service account:
```json
  "ec2:DescribeImages",
  "ec2:GetInstanceTypesFromInstanceRequirements",
```
specified in https://github.com/terraform-aws-modules/terraform-aws-iam/blob/master/modules/iam-role-for-service-accounts-eks/policies.tf#L52

This prevents cluster autocaler to retrieve information from AWS to scale requirements based ASG-s. It fails with similar logs:
```log
E0313 11:50:19.841266 1 aws_wrapper.go:723] Failed to query instance requirements for ASG xxxx-2023031308565983410000000a: UnauthorizedOperation: You are not authorized to perform this operation.
```

## Describe the solution you'd like.
Add 
```json
  "ec2:DescribeImages",
  "ec2:GetInstanceTypesFromInstanceRequirements",
```
permissions to EKS Autoscaler IAM role

## Additional context
PR for Autoscale that enables instance requirements support is here: https://github.com/kubernetes/autoscaler/pull/4588/files

**Comment by antonbabenko** [comment]
This issue has been resolved in version 5.15.0 :tada:

**Comment by bryantbiggs** [comment]
resolved in #356
